---
result: FAILURE
timestamp: 2016-04-22 21:41:54 UTC
duration: 4263.794
triggeredBy: tyler-ball
url: http://manhattan.ci.chef.co/job/chefdk-trigger-ad_hoc/216/
version: 0.13.23+20160422214209
git_ref: unified_workflow_spike
git_commit: 00fefa1366ea3ef72fc97f0bf940af73db4ba6fc
failures:
  chefdk-test code - chef-acceptance[top-cookbooks (provision),top-cookbooks (force-destroy)]: acceptance
  chefdk-test code - chef verify[opscode-pushy-client]: debian,el,mac_os_x,ubuntu
stages:
  chefdk-test:
    result: FAILURE
    failures:
      code - chef-acceptance[top-cookbooks (provision),top-cookbooks (force-destroy)]: acceptance
      code - chef verify[opscode-pushy-client]: debian,el,mac_os_x,ubuntu
    timestamp: 2016-04-22 22:32:01 UTC
    duration: 1275.691
    url: http://manhattan.ci.chef.co/job/chefdk-test/130/
    parameters:
      OMNIBUS_BUILD_VERSION: 0.13.23+20160422214209
      DELIVERY_CHANGE: ''
      DELIVERY_SHA: ''
      GIT_REF: unified_workflow_spike
      APPEND_TIMESTAMP: true
      EXPIRE_CACHE: false
      RELEASE_BUILD: 'false'
      PIPELINE_NAME: chefdk
      GIT_COMMIT: 00fefa1366ea3ef72fc97f0bf940af73db4ba6fc
      PROJECT: chefdk
      PROMOTE: 'false'
      PIPELINE_TRIGGER_BUILD_NUMBER: '216'
      ADHOC_BUILD: 'true'
      GIT_BRANCH: origin/unified_workflow_spike
      PIPELINE_TRIGGER_JOB_NAME: chefdk-trigger-ad_hoc
      PLATFORM_MAPPINGS_JSON: |-
        {
          "build_to_test": {
            "debian-6": [
              "debian-6",
              "debian-7",
              "debian-8"
            ],
            "ubuntu-12.04": [
              "ubuntu-12.04",
              "ubuntu-14.04",
              "acceptance"
            ],
            "el-6": [
              "el-6"
            ],
            "el-7": [
              "el-7"
            ],
            "mac_os_x-10.9": [
              "mac_os_x-10.9",
              "mac_os_x-10.10",
              "mac_os_x-10.11"
            ],
            "windows-2008r2": [
              "windows-2008r2",
              "windows-2012",
              "windows-2012r2"
            ]
          },
          "test_to_build": {
            "debian-6": "debian-6",
            "debian-7": "debian-6",
            "debian-8": "debian-6",
            "ubuntu-12.04": "ubuntu-12.04",
            "ubuntu-14.04": "ubuntu-12.04",
            "acceptance": "ubuntu-12.04",
            "el-6": "el-6",
            "el-7": "el-7",
            "mac_os_x-10.9": "mac_os_x-10.9",
            "mac_os_x-10.10": "mac_os_x-10.9",
            "mac_os_x-10.11": "mac_os_x-10.9",
            "windows-2008r2": "windows-2008r2",
            "windows-2012": "windows-2008r2",
            "windows-2012r2": "windows-2008r2"
          }
        }
      PIPELINE_BUILD_JOB_NUMBER: '316'
      PIPELINE_BUILD_JOB_NAME: chefdk-build
      PIPELINE_RUN_FINGERPRINT: jenkins-chefdk-trigger-ad_hoc-216/216/2016-04-22_21-41-54
    runs:
      acceptance:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 83.674
        builtOn: chefdk-ubuntu-14.04-acceptance-6eb047
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=acceptance,project=chefdk,role=tester/130/
        failureCause:
          cause: chef-acceptance[top-cookbooks (provision),top-cookbooks (force-destroy)]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            526: "TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:17 +0000] \n        \n
              \       ================================================================================\n
              \       Error executing action `sync` on resource 'git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]'\n
              \       ================================================================================\n
              \       \n        Mixlib::ShellOut::ShellCommandFailed\n        ------------------------------------\n
              \       Expected process to exit with [0], but received '128'\n        ----
              Begin output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n        STDOUT: \n        STDERR: fatal: unable to
              access 'https://github.com/chef-cookbooks/docker.git/': error setting
              certificate verify locations:\n          CAfile: /opt/chefdk/embedded/ssl/certs\n
              \         CApath: none\n        ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n        Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n        \n        Resource Declaration:\n
              \       ---------------------\n        # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb\n
              \       \n         36:     git repository_root do\n         37:       repository
              new_resource.repository\n         38:       branch new_resource.branch\n
              \        39:     end\n         40: \n        \n        Compiled Resource:\n
              \       ------------------\n        # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb:36:in
              `block in <class:CookbookKitchen>'\n        \n        git(\"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker\")
              do\n          action [:sync]\n          retries 0\n          retry_delay
              2\n          default_guard_interpreter :default\n          destination
              \"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker\"\n
              \         enable_checkout true\n          revision \"master\"\n          remote
              \"origin\"\n          checkout_branch \"deploy\"\n          declared_type
              :git\n          cookbook_name \"acceptance-cookbook\"\n          repository
              \"https://github.com/chef-cookbooks/docker.git\"\n        end\n        \n
              \       Platform:\n        ---------\n        x86_64-linux\n        \n
              \     \n      ================================================================================\n
              \     Error executing action `run` on resource 'cookbook_kitchen[converge
              docker]'\n      ================================================================================\n
              \     \n      Mixlib::ShellOut::ShellCommandFailed\n      ------------------------------------\n
              \     git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]
              (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb
              line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected
              process to exit with [0], but received '128'\n      ---- Begin output
              of git ls-remote \"https://github.com/chef-cookbooks/docker.git\" \"master*\"
              ----\n      STDOUT: \n      STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/':
              error setting certificate verify locations:\n        CAfile: /opt/chefdk/embedded/ssl/certs\n
              \       CApath: none\n      ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n      Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n      \n      Resource Declaration:\n      ---------------------\n
              \     # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/coo\nTOP-COOKBOOKS::PROVISION::[2016-04-22
              22:33:17 +0000] kbooks/acceptance-cookbook/libraries/top_cookbooks.rb\n
              \     \n"
            629: "      \n    \n    ================================================================================\n
              \   Error executing action `run` on resource 'top_cookbooks[converge]'\n
              \   ================================================================================\n
              \   \n    Mixlib::ShellOut::ShellCommandFailed\n    ------------------------------------\n
              \   cookbook_kitchen[converge docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb
              line 7) had an error: Mixlib::ShellOut::ShellCommandFailed: git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]
              (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb
              line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected
              process to exit with [0], but received '128'\n    ---- Begin output
              of git ls-remote \"https://github.com/chef-cookbooks/docker.git\" \"master*\"
              ----\n    STDOUT: \n    STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/':
              error setting certificate verify locations:\n      CAfile: /opt/chefdk/embedded/ssl/certs\n
              \     CApath: none\n    ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n    Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n    \n    Resource Declaration:\n    ---------------------\n
              \   # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/provision.rb\n
              \   \n      1: top_cookbooks \"converge\"\n    \n    Compiled Resource:\n
              \   ------------------\n    # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/provision.rb:1:in
              `from_file'\n    \n    top_cookbooks(\"converge\") do\n      action
              [:run]\n      updated true\n      updated_by_last_action true\n      retries
              0\n      retry_delay 2\n      default_guard_interpreter :default\n      declared_type
              :top_cookbooks\n      cookbook_name \"acceptance-cookbook\"\n      recipe_name
              \"provision\"\n      command \"converge\"\n    end\n    \n    Platform:\n
              \   ---------\n    x\nTOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:17
              +0000] 86_64-linux\n    \n"
            679: |2

              Running handlers:
              [2016-04-22T22:33:18+00:00] ERROR: Running exception handlers
              Running handlers complete
              [2016-04-22T22:33:18+00:00] ERROR: Exception handlers complete
              Chef Client failed. 1 resources updated in 03 seconds
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] ERROR: Failed to post audit report to server. Saving report to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/failed-audit-data.json
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] FATAL: Stacktrace dumped to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/chef-stacktrace.out
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] ERROR: Found 1 errors, they are stored in the backtrace
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
              CHEF-ACCEPTANCE::[2016-04-22 22:33:18 +0000] Encountered an error running the recipe provision: Expected process to exit with [0], but received '1'
              ---- Begin output of chef-client -z -c /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/config.rb --force-formatter -j /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/dna.json -o acceptance-cookbook::provision --no-color ----
            708: "      * git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]
              action sync\n        \n        ================================================================================\n
              \       Error executing action `sync` on resource 'git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]'\n
              \       ================================================================================\n
              \       \n        Mixlib::ShellOut::ShellCommandFailed\n        ------------------------------------\n
              \       Expected process to exit with [0], but received '128'\n        ----
              Begin output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n        STDOUT: \n        STDERR: fatal: unable to
              access 'https://github.com/chef-cookbooks/docker.git/': error setting
              certificate verify locations:\n          CAfile: /opt/chefdk/embedded/ssl/certs\n
              \         CApath: none\n        ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n        Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n        \n        Resource Declaration:\n
              \       ---------------------\n        # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb\n
              \       \n         36:     git repository_root do\n         37:       repository
              new_resource.repository\n         38:       branch new_resource.branch\n
              \        39:     end\n         40: \n        \n        Compiled Resource:\n
              \       ------------------\n        # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb:36:in
              `block in <class:CookbookKitchen>'\n        \n        git(\"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker\")
              do\n          action [:sync]\n          retries 0\n          retry_delay
              2\n          default_guard_interpreter :default\n          destination
              \"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker\"\n
              \         enable_checkout true\n          revision \"master\"\n          remote
              \"origin\"\n          checkout_branch \"deploy\"\n          declared_type
              :git\n          cookbook_name \"acceptance-cookbook\"\n          repository
              \"https://github.com/chef-cookbooks/docker.git\"\n        end\n        \n
              \       Platform:\n        ---------\n        x86_64-linux\n        \n
              \     \n      ================================================================================\n
              \     Error executing action `run` on resource 'cookbook_kitchen[converge
              docker]'\n      ================================================================================\n
              \     \n      Mixlib::ShellOut::ShellCommandFailed\n      ------------------------------------\n
              \     git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]
              (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb
              line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected
              process to exit with [0], but received '128'\n      ---- Begin output
              of git ls-remote \"https://github.com/chef-cookbooks/docker.git\" \"master*\"
              ----\n      STDOUT: \n      STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/':
              error setting certificate verify locations:\n        CAfile: /opt/chefdk/embedded/ssl/certs\n
              \       CApath: none\n      ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n      Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n      \n      Resource Declaration:\n      ---------------------\n
              \     # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb\n
              \     \n        7:     cookbook_kitchen \"#{command} docker\" do\n        8:
              \    end\n        9: \n      \n      Compiled Resource:\n      ------------------\n
              \     # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb:7:in
              `block in <class:TopCookbooks>'\n      \n      cookbook_kitchen(\"converge
              docker\") do\n        action [:run]\n        updated true\n        updated_by_last_action
              true\n        retries 0\n        retry_delay 2\n        default_guard_interpreter
              :default\n        declared_type :cookbook_kitchen\n        cookbook_name
              \"acceptance-cookbook\"\n        repository \"https://github.com/chef-cookbooks/docker.git\"\n
              \       test_cookbook \"docker\"\n        repository_root \"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker\"\n
              \       branch \"master\"\n        command \"converge\"\n        kitchen_options
              \"-c\"\n        kitchen_dir \"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker/\"\n
              \       chef_channel \"unstable\"\n        chef_version \"latest\"\n
              \       env {\"BUNDLE_GEMFILE\"=>\"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/Gemfile\",
              \"KITCHEN_YAML\"=>\"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.kitchen.docker.yml\"}\n
              \     end\n      \n      Platform:\n      ---------\n      x86_64-linux\n
              \     \n    \n    ================================================================================\n
              \   Error executing action `run` on resource 'top_cookbooks[converge]'\n
              \   ================================================================================\n
              \   \n    Mixlib::ShellOut::ShellCommandFailed\n    ------------------------------------\n
              \   cookbook_kitchen[converge docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb
              line 7) had an error: Mixlib::ShellOut::ShellCommandFailed: git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]
              (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb
              line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected
              process to exit with [0], but received '128'\n    ---- Begin output
              of git ls-remote \"https://github.com/chef-cookbooks/docker.git\" \"master*\"
              ----\n    STDOUT: \n    STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/':
              error setting certificate verify locations:\n      CAfile: /opt/chefdk/embedded/ssl/certs\n
              \     CApath: none\n    ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n    Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n    \n    Resource Declaration:\n    ---------------------\n
              \   # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/provision.rb\n
              \   \n      1: top_cookbooks \"converge\"\n    \n    Compiled Resource:\n
              \   ------------------\n    # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/provision.rb:1:in
              `from_file'\n    \n    top_cookbooks(\"converge\") do\n      action
              [:run]\n      updated true\n      updated_by_last_action true\n      retries
              0\n      retry_delay 2\n      default_guard_interpreter :default\n      declared_type
              :top_cookbooks\n      cookbook_name \"acceptance-cookbook\"\n      recipe_name
              \"provision\"\n      command \"converge\"\n    end\n    \n    Platform:\n
              \   ---------\n    x86_64-linux\n    \nStarting audit phase\n\n"
            859: "\nRunning handlers:\n[2016-04-22T22:33:18+00:00] ERROR: Running
              exception handlers\nRunning handlers complete\n[2016-04-22T22:33:18+00:00]
              ERROR: Exception handlers complete\nChef Client failed. 1 resources
              updated in 03 seconds\n[2016-04-22T22:33:18+00:00] ERROR: Failed to
              post audit report to server. Saving report to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/failed-audit-data.json\n[2016-04-22T22:33:18+00:00]
              FATAL: Stacktrace dumped to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/chef-stacktrace.out\n[2016-04-22T22:33:18+00:00]
              FATAL: Please provide the contents of the stacktrace.out file if you
              file a bug report\n[2016-04-22T22:33:18+00:00] ERROR: Found 1 errors,
              they are stored in the backtrace\n[2016-04-22T22:33:18+00:00] FATAL:
              Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully
              (exit code 1)\nSTDERR: \n---- End output of chef-client -z -c /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/config.rb
              --force-formatter -j /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/dna.json
              -o acceptance-cookbook::provision --no-color ----\nRan chef-client -z
              -c /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/config.rb
              --force-formatter -j /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/dna.json
              -o acceptance-cookbook::provision --no-color returned 1\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/mixlib-shellout-2.2.6/lib/mixlib/shellout.rb:289:in
              `invalid!'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/mixlib-shellout-2.2.6/lib/mixlib/shellout.rb:276:in
              `error!'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/chef_runner.rb:40:in
              `block in run!'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler.rb:219:in
              `block in with_clean_env'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler.rb:199:in
              `with_original_env'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler.rb:205:in
              `with_clean_env'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/chef_runner.rb:35:in
              `run!'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:144:in
              `run_command'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:111:in
              `block in run_suite'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:109:in
              `each'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:109:in
              `run_suite'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:75:in
              `block (2 levels) in start_worker'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:73:in
              `loop'\n/opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:73:in
              `block in start_worker'\nCHEF-ACCEPTANCE::[2016-04-22 22:33:18 +0000]
              --force-destroy specified so attempting to run the destroy recipe\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:18 +0000] Initialized [.acceptance_logs/top-cookbooks/destroy.log]
              logger...\n"
            924: "TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:24 +0000] \n        \n
              \       ================================================================================\n
              \       Error executing action `sync` on resource 'git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]'\n
              \       ================================================================================\n
              \       \n        Mixlib::ShellOut::ShellCommandFailed\n        ------------------------------------\n
              \       Expected process to exit with [0], but received '128'\n        ----
              Begin output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n        STDOUT: \n        STDERR: fatal: unable to
              access 'https://github.com/chef-cookbooks/docker.git/': error setting
              certificate verify locations:\n          CAfile: /opt/chefdk/embedded/ssl/certs\n
              \         CApath: none\n        ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n        Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n        \n        Resource Declaration:\n
              \       ---------------------\n        # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb\n
              \       \n         36:     git repository_root do\n         37:       repository
              new_resource.repository\n         38:       branch new_resource.branch\n
              \        39:     end\n         40: \n        \n        Compiled Resource:\n
              \       ------------------\n        # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb:36:in
              `block in <class:CookbookKitchen>'\n        \n        git(\"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker\")
              do\n          action [:sync]\n          retries 0\n          retry_delay
              2\n          default_guard_interpreter :default\n          destination
              \"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker\"\n
              \         enable_checkout true\n          revision \"master\"\n          remote
              \"origin\"\n          checkout_branch \"deploy\"\n          declared_type
              :git\n          cookbook_name \"acceptance-cookbook\"\n          repository
              \"https://github.com/chef-cookbooks/docker.git\"\n        end\n        \n
              \       Platform:\n        ---------\n        x86_64-linux\n        \n
              \     \n      ================================================================================\n
              \     Error executing action `run` on resource 'cookbook_kitchen[destroy
              docker]'\n      ================================================================================\n
              \     \n      Mixlib::ShellOut::ShellCommandFailed\n      ------------------------------------\n
              \     git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]
              (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb
              line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected
              process to exit with [0], but received '128'\n      ---- Begin output
              of git ls-remote \"https://github.com/chef-cookbooks/docker.git\" \"master*\"
              ----\n      STDOUT: \n      STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/':
              error setting certificate verify locations:\n        CAfile: /opt/chefdk/embedded/ssl/certs\n
              \       CApath: none\n      ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n      Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n      \n      Resource Declaration:\n      ---------------------\n
              \     # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cook\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:24 +0000] books/acceptance-cookbook/libraries/top_cookbooks.rb\n
              \     \n"
            1025: "      \n    \n    ================================================================================\n
              \   Error executing action `run` on resource 'top_cookbooks[destroy]'\n
              \   ================================================================================\n
              \   \n    Mixlib::ShellOut::ShellCommandFailed\n    ------------------------------------\n
              \   cookbook_kitchen[destroy docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb
              line 7) had an error: Mixlib::ShellOut::ShellCommandFailed: git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]
              (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb
              line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected
              process to exit with [0], but received '128'\n    ---- Begin output
              of git ls-remote \"https://github.com/chef-cookbooks/docker.git\" \"master*\"
              ----\n    STDOUT: \n    STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/':
              error setting certificate verify locations:\n      CAfile: /opt/chefdk/embedded/ssl/certs\n
              \     CApath: none\n    ---- End output of git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" ----\n    Ran git ls-remote \"https://github.com/chef-cookbooks/docker.git\"
              \"master*\" returned 128\n    \n    Resource Declaration:\n    ---------------------\n
              \   # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/destroy.rb\n
              \   \n      1: top_cookbooks \"destroy\"\n    \n    Compiled Resource:\n
              \   ------------------\n    # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/destroy.rb:1:in
              `from_file'\n    \n    top_cookbooks(\"destroy\") do\n      action [:run]\n
              \     retries 0\n      retry_delay 2\n      default_guard_interpreter
              :default\n      declared_type :top_cookbooks\n      cookbook_name \"acceptance-cookbook\"\n
              \     recipe_name \"destroy\"\n      command \"destroy\"\n    end\n
              \   \n    Platform:\n    ---------\n    x86_64-linux\n    \nStarting
              audit phase\nTOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] \n"
            1074: "TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] Running handlers:\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:25 +0000] \nTOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000]
              [2016-04-22T22:33:25+00:00] ERROR: Running exception handlers\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:25 +0000] Running handlers complete\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:25 +0000] [2016-04-22T22:33:25+00:00] ERROR: Exception handlers
              complete\nTOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] Chef Client
              failed. 0 resources updated in 03 seconds\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:25 +0000] \nTOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000]
              [2016-04-22T22:33:25+00:00] ERROR: Failed to post audit report to server.
              Saving report to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/failed-audit-data.json\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:25 +0000] [2016-04-22T22:33:25+00:00] FATAL: Stacktrace dumped
              to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/chef-stacktrace.out\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:25 +0000] [2016-04-22T22:33:25+00:00] FATAL: Please provide the
              contents of the stacktrace.out file if you file a bug report\nTOP-COOKBOOKS::DESTROY::[2016-04-22
              22:33:25 +0000] [2016-04-22T22:33:25+00:00] ERROR: Found 1 errors, they
              are stored in the backtrace\nTOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25
              +0000] [2016-04-22T22:33:25+00:00] FATAL: Chef::Exceptions::ChildConvergeError:
              Chef run process exited unsuccessfully (exit code 1)\nCHEF-ACCEPTANCE::[2016-04-22
              22:33:25 +0000] \nCHEF-ACCEPTANCE::[2016-04-22 22:33:25 +0000] chef-acceptance
              run failed\n"
            1091: |
              CHEF-ACCEPTANCE::[2016-04-22 22:33:25 +0000] | top-cookbooks | Total         | 00:00:14 | Y     |
              CHEF-ACCEPTANCE::[2016-04-22 22:33:25 +0000] | Run           | Total         | 00:00:14 | Y     |
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef-acceptance:
            - top-cookbooks (provision)
            - top-cookbooks (force-destroy)
        acceptanceTiming:
        - top-cookbooks:
            provision: '00:00:04'
            force-destroy: '00:00:04'
            Total: '00:00:14'
          Run:
            Total: '00:00:14'
        omnibusTiming: {}
      debian-6:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 710.474
        builtOn: chefdk-debian-6-tester-31349b
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=debian-6,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            509: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            530: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            664: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      debian-7:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 483.916
        builtOn: chefdk-debian-7-tester-6585d4
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=debian-7,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            494: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            515: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            649: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      debian-8:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 558.599
        builtOn: chefdk-debian-8-tester-b7ac3b
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=debian-8,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            492: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            513: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            647: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      el-6:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 1274.999
        builtOn: chefdk-rhel-6-tester-42bd5e
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=el-6,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            481: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            502: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            652: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      el-7:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 434.766
        builtOn: chefdk-rhel-7-tester-ce15ff
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=el-7,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            488: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            509: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            659: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      mac_os_x-10.10:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 316.743
        builtOn: chefdk-mac_os_x-10.10-tester-f8e0cd
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=mac_os_x-10.10,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            388: "/dev/disk1            GUID_partition_scheme            \n/dev/disk1s1
              \         Apple_HFS                        /Volumes/chefdk\nfind: /Volumes/chefdk/.Trashes:
              Permission denied\ninstaller: Package name is Chef Development Kit\ninstaller:
              Installing at base path /\n"
            443: |
              For non-Windows platforms, make sure libzmq is located in this search path:
              ["/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/../../ext/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/embedded/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/languages/ruby/2.1.5/bin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/libzmq.dylib", "/usr/local/sbin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/usr/sbin/libzmq.dylib", "/usr/bin/libzmq.dylib", "/sbin/libzmq.dylib", "/bin/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/opt/local/lib/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/usr/lib64/libzmq.dylib"]
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:54:in `rescue in <module:LibZMQ>': The libzmq library (or DLL) could not be loaded (LoadError)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:9:in `<module:LibZMQ>'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:6:in `<top (required)>'
            536: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            557: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            691: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      mac_os_x-10.11:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 235.321
        builtOn: chefdk-mac_os_x-10.11-tester-fb35b7
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=mac_os_x-10.11,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            388: "/dev/disk1            GUID_partition_scheme            \n/dev/disk1s1
              \         Apple_HFS                        /Volumes/chefdk\nfind: /Volumes/chefdk/.Trashes:
              Permission denied\ninstaller: Package name is Chef Development Kit\ninstaller:
              Installing at base path /\n"
            443: |
              For non-Windows platforms, make sure libzmq is located in this search path:
              ["/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/../../ext/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/embedded/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/languages/ruby/2.1.5/bin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/libzmq.dylib", "/usr/local/sbin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/usr/sbin/libzmq.dylib", "/usr/bin/libzmq.dylib", "/sbin/libzmq.dylib", "/bin/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/opt/local/lib/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/usr/lib64/libzmq.dylib"]
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:54:in `rescue in <module:LibZMQ>': The libzmq library (or DLL) could not be loaded (LoadError)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:9:in `<module:LibZMQ>'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:6:in `<top (required)>'
            536: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            557: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            691: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      mac_os_x-10.9:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 298.693
        builtOn: chefdk-mac_os_x-10.9-tester-8a14ce
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=mac_os_x-10.9,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            388: "/dev/disk1            GUID_partition_scheme            \n/dev/disk1s1
              \         Apple_HFS                        /Volumes/chefdk\nfind: /Volumes/chefdk/.Trashes:
              Permission denied\ninstaller: Package name is Chef Development Kit\ninstaller:
              Installing at base path /\n"
            443: |
              For non-Windows platforms, make sure libzmq is located in this search path:
              ["/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/../../ext/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/embedded/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/languages/ruby/2.1.5/bin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/libzmq.dylib", "/usr/local/sbin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/usr/sbin/libzmq.dylib", "/usr/bin/libzmq.dylib", "/sbin/libzmq.dylib", "/bin/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/opt/local/lib/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/usr/lib64/libzmq.dylib"]
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:54:in `rescue in <module:LibZMQ>': The libzmq library (or DLL) could not be loaded (LoadError)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:9:in `<module:LibZMQ>'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:6:in `<top (required)>'
            536: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            557: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            691: |
              Verification of component 'chef-sugar' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      ubuntu-12.04:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 448.181
        builtOn: chefdk-ubuntu-12.04-tester-d64e0a
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=ubuntu-12.04,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            494: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            515: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            649: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      ubuntu-14.04:
        result: FAILURE
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 668.92
        builtOn: chefdk-ubuntu-14.04-tester-48c4bf
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=ubuntu-14.04,project=chefdk,role=tester/130/
        failureCause:
          cause: chef verify[opscode-pushy-client]
          jenkinsBuildStep: Invoke XShell command
          category: code
          suspiciousBlocks:
            496: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            517: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            651: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
          tests:
            chef verify:
            - opscode-pushy-client
        omnibusTiming: {}
      windows-2008r2-i386:
        result: SUCCESS
        timestamp: 2016-04-22 22:32:01 UTC
        duration: 579.634
        builtOn: chefdk-windows-2008r2-i386-tester-e50cbf
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=i386,platform=windows-2008r2,project=chefdk,role=tester/130/
        omnibusTiming: {}
  chefdk-build:
    result: SUCCESS
    timestamp: 2016-04-22 21:42:09 UTC
    duration: 2983.691
    url: http://manhattan.ci.chef.co/job/chefdk-build/316/
    parameters:
      GIT_REF: unified_workflow_spike
      APPEND_TIMESTAMP: true
      DELIVERY_CHANGE: ''
      DELIVERY_SHA: ''
      EXPIRE_CACHE: false
      RELEASE_BUILD: 'false'
      PIPELINE_NAME: chefdk
      GIT_COMMIT: 00fefa1366ea3ef72fc97f0bf940af73db4ba6fc
      PROJECT: chefdk
      PROMOTE: 'false'
      PIPELINE_TRIGGER_BUILD_NUMBER: '216'
      ADHOC_BUILD: 'true'
      GIT_BRANCH: origin/unified_workflow_spike
      PIPELINE_TRIGGER_JOB_NAME: chefdk-trigger-ad_hoc
    runs:
      debian-6:
        result: SUCCESS
        timestamp: 2016-04-22 21:42:09 UTC
        duration: 2537.88
        builtOn: chefdk-debian-6-builder-2ccc58
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=debian-6,project=chefdk,role=builder/316/
        omnibusTiming: {}
      el-6:
        result: SUCCESS
        timestamp: 2016-04-22 21:42:09 UTC
        duration: 2978.86
        builtOn: chefdk-rhel-6-builder-02167c
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=el-6,project=chefdk,role=builder/316/
        omnibusTiming: {}
      el-7:
        result: SUCCESS
        timestamp: 2016-04-22 21:42:09 UTC
        duration: 2332.059
        builtOn: chefdk-rhel-7-builder-b67c93
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=el-7,project=chefdk,role=builder/316/
        omnibusTiming: {}
      mac_os_x-10.9:
        result: SUCCESS
        timestamp: 2016-04-22 21:42:09 UTC
        duration: 985.144
        builtOn: chefdk-mac_os_x-10.9-builder-d736d1
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=mac_os_x-10.9,project=chefdk,role=builder/316/
        omnibusTiming: {}
      ubuntu-12.04:
        result: SUCCESS
        timestamp: 2016-04-22 21:42:09 UTC
        duration: 2490.066
        builtOn: chefdk-ubuntu-12.04-builder-efc2e0
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=ubuntu-12.04,project=chefdk,role=builder/316/
        omnibusTiming: {}
      windows-2008r2-i386:
        result: SUCCESS
        timestamp: 2016-04-22 21:42:09 UTC
        duration: 1784.22
        builtOn: chefdk-windows-2008r2-i386-builder-dcb6f4
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=i386,platform=windows-2008r2,project=chefdk,role=builder/316/
        omnibusTiming: {}
  chefdk-trigger-ad_hoc:
    result: SUCCESS
    timestamp: 2016-04-22 21:41:54 UTC
    duration: 4.412
    url: http://manhattan.ci.chef.co/job/chefdk-trigger-ad_hoc/216/
    causes:
    - shortDescription: Started by user Tyler Ball
      userId: tyler-ball
      userName: Tyler Ball
    parameters:
      GIT_REF: unified_workflow_spike
      EXPIRE_CACHE: false
    runs: {}
